LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
```

Comment:
We need to fill this... From what I've seen in other repos: Copyright (c) 2023 Crown Copyright NHS England.

Reply:
Done :heavy_check_mark: Updated to follow definition in https://github.com/nhsx/open-source-policy/blob/main/open-source-policy.md.